docs/torch/link_pred.rst

```diff
@@ -0,0 +1,371 @@
+****************************
+Custom Link Prediction Model
+****************************
+
+In this guide we build a Link Prediction model to predict whether there is a link between any two nodes of the `caveman dataset <https://networkx.org/documentation/stable/reference/generated/networkx.generators.community.connected_caveman_graph.html?highlight=connected_caveman_graph#networkx.generators.community.connected_caveman_graph>`_.
+
+If you are interested in the high level DeepGNN training flow, see the `overiew guide <quickstart.html>`_. This guide focuses on building a custom model.
+
+
+Generate Dataset
+================
```

Comment:
This is already covered by graph_engine/from_networkx, maybe we move all the explanations on how to generate a dataset from this file to graph_engine/ and keep this code with a link. This way this link_pred file will just be a description on how to create a custom model? Same for TF

Reply:
I am converting markdown to rST, not fixing the existing docs.